akka-http-core/src/main/resources/reference.conf

```diff
@@ -431,6 +431,14 @@ akka.http {
 
       # Enable verbose debug logging for all ingoing and outgoing frames
       log-frames = false
+
+      # When there is no data transmitted, send a HTTP/2 ping frame with this interval to make sure the connection is
```

Comment:
Perhaps allow those to be configured separately for the server and the client side?

Reply:
Yeah, it recently struck me that it should be for both client and server, I started out only thinking of it for client side.